sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -20,18 +20,14 @@
           placeholder="yyyy-MM-dd"
           v-model="formModel.effectiveEndDate"
           variant="outlined"
-          :rules="[
-            (v) =>
-              v.match(/^\d{4}-\d{2}-\d{2}$/) ||
-              'Effective end date is not in right format',
-          ]"
+          :rules="[(v) => checkStringDateFormatRule(v)]"
```

Comment:
If I am not wrong it can be simplified as below. Can you please give it a try? If works we can define it as a default way to do it.
```ts
:rules="[checkStringDateFormatRule]"
```

Reply:
it works, i changed all